### DISCUSSION TITLE
Can't draw Currency symbols

### DISCUSSION THREAD

**Discussion by yehuda-simtex** [SixLabors/ImageSharp.Drawing#296] [Q&A]
I am using Google Fonts 'Arimo' to try and write a currency symbol, for example: ₪
The Glyph does not come up. Other currency codes do come out.
text: "USD: $; EUR: €; ILS: ₪; GBP: £"
Output: 
![image](https://github.com/SixLabors/ImageSharp.Drawing/assets/145898435/8b5816e4-0885-4530-bd8d-cb3e88802b77)

Any ideas?

**Comment by JimBobSquarePants** [comment]
That would suggest to me that the font does not contain that particular symbol. You’ll need a fallback font.

**Comment by yehuda-simtex** [comment]
If you visit the [test page](https://fonts.google.com/specimen/Arimo/tester), it will render the symbol properly.

**Reply by tocsoft** [reply]
That pages is using the browsers rendering engine to render the font, browsers automaticaly render using one or more fallback fonts to prevent issues where font don't have particular glyphs (as in this case)... you can use a tool like https://fontdrop.info (I recommend using the static  `Arimo-Regular.ttf` file downloaded from google link you provided, I had issues getting the the variable ones working with that site) to verify what glyphs are present in the file.

**Note:** for tabs other than the **Glyphs** tab fontdrop is also effected by the issue where the browser auto applying a fallback font during rendering.

**Reply by yehuda-simtex** [reply]
Issue resolved - the download from Google was wrong. Once this was fixed, the symbol did render correctly.
